### DISCUSSION TITLE
Compatible with flysystem

### DISCUSSION THREAD

**Discussion by Snowbaha** [supabase/supabase#11423] [Questions] [answered]
Hello,

I am trying to use Supabase Storage with a service like https://flysystem.thephpleague.com/docs/adapter/aws-s3-v3/
for a Symfony application.
I tried the ASW3 method in https://github.com/thephpleague/flysystem-bundle/blob/3.x/docs/2-cloud-storage-providers.md#aws-sdk-s3

But it is not working as expected. Do you have any idea how to use it?

**Comment by GaryAustin1** [comment, marked as answer]
If you are trying to access the Supabase hosted Storage system directly like you would an s3 bucket you can't.  You need to go thru the Storage API calls.  There is no s3 bucket (credentials or otherwise) available for each instance.
